tarn/pool/pickle_key.py

```diff
@@ -72,7 +88,8 @@ def write(self, key: ProxyKey, value: Any, *, error: bool = True, labels: MaybeL
         # we want a reproducible mapping each time
         logger.info('Saving to index %s', digest)
         try:
-            with self.index.write(digest, BytesIO(json.dumps(mapping, sort_keys=True).encode()), labels=None) as written:
+            with self.index.write(digest, BytesIO(json.dumps(mapping, sort_keys=True).encode()),
+                                  labels=None) as written:
```

Comment:
is your formatter set up to allow 120 characters per line?

Reply:
Yep